packages/create-react-native-library/templates/example/example/android/app/build.gradle

```diff
@@ -77,7 +77,7 @@ import com.android.build.OutputFile
 
 project.ext.react = [
     enableHermes: false,  // clean and rebuild if changing
-    entryFile: "index.tsx",
+    entryFile: "index.js",
```

Comment:
Isn't our entry file `index.tsx`?

Reply:
I needed to change it, bc iOS did not build correctly. I did not know, where to change entryFile for iOS